plex/api.py

```diff
@@ -525,6 +526,7 @@ def fillVideoInfos(
                 'playcount': plexItem.viewCount,
                 'lastplayed': Api.convertDateTimeToDbDateTime(plexItem.lastViewedAt),
             })
+            info['tag'].append(f"PlexLibrary:{plexItem.librarySectionID}")
```

Comment:
What does the `librarySectionID` look like? Is it a number or a string? In `mediaimporter.emby` we simply tag all items with the name of the library they come from.

Reply:
librarySectionId is a number representing the library. the plexItem doesn't have the name of the library, just the number. I'll do a little more digging and see how to map that back to the library name and tag that instead.

Reply:
Looks like I can get it from the librarySection if I pass that through. Will need to get rebased onto the delta_sync branch where the toFileItem happens within the section loop to pass it through easily. I'll work on that.

Reply:
@Montellese Take a look now, got it updated to use the library name by patching plexapi to include that in the Video class. Confirmed in the API that in all places the librarySectionID is present the librarySectionTitle is as well. Another proposal I'll include to upstream.